.github/run_browserstack_acceptance.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+export BROWSERSTACK_USE_AUTOMATE="1"
+export BROWSERSTACK_PROJECT_NAME="Answers SDK"
+export BROWSERSTACK_BUILD_ID="${GITHUB_REF_NAME} - ${GITHUB_RUN_ID} + ${CIRCLE_BUILD_NUM}"
```

Comment:
Why do we use `CIRCLE_BUILD_NUM` here?

Reply:
oh shoot this was from debugging when I wanted to differentiate between the two in the browserstack automate UI, I'll remove!